src/main/java/com/entropyteam/entropay/employees/controllers/ContractController.java

```diff
@@ -24,4 +27,12 @@ public ContractController(ContractService contractService) {
     public ResponseEntity<ContractDto> modifyStatus(@PathVariable UUID id, @RequestParam boolean active) {
         return ResponseEntity.ok(contractService.modifyStatus(id, active));
     }
+
+    @GetMapping("/contractTypes")
```

Comment:
ohhh this opens a beautiful debate... should we use a hyphen, underscore, or camelCase for the URLs? 

Not right answer, but I personally prefer hyphen

one SO post: https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris

Reply:
haha, I agree with hyphen,  `/contract-types`

Reply:
Done!